application/helpers/export_helper.php

```diff
@@ -1341,6 +1341,29 @@ function quexml_set_default_value(&$element, $iResponseID, $qid, $iSurveyID, $fi
     }
 }
 
+/**
+ * Format defaultValue of Date/Time questions according to question date format
+ *
+ * @param mixed $element DOM element with the date to change
+ * @param int $qid The qid of the question
+ * @param int $iSurveyID The survey id
+ */
+function quexml_reformat_date(&$element, $qid, $iSurveyID)
```

Comment:
`&` not needed

Reply:
I prefer to keep it as the other methods for consistency if thats fine

Reply:
I actually consider this a bug in the function signature, because it communicates the wrong intent to the code reader. But I think I'll fix it (and in the other places where it's not needed).